tools/js-sdk-release-tools/src/common/changlog/automaticGenerateChangeLogAndBumpVersion.ts

```diff
@@ -50,62 +50,90 @@ export async function generateChangelogAndBumpVersion(packageFolderPath: string)
         try {
             shell.mkdir(path.join(packageFolderPath, 'changelog-temp'));
             shell.cd(path.join(packageFolderPath, 'changelog-temp'));
-            shell.exec(`npm pack ${packageName}@${stableVersion}`, { silent: true});
+            shell.exec(`npm pack ${packageName}@${stableVersion}`, { silent: true });
             const files = shell.ls('*.tgz');
             shell.exec(`tar -xzf ${files[0]}`);
             shell.cd(packageFolderPath);
 
             // only track2 sdk includes sdk-type with value mgmt
-            const sdkType = JSON.parse(fs.readFileSync(path.join(packageFolderPath, 'changelog-temp', 'package', 'package.json'), {encoding: 'utf-8'}))['sdk-type'];
+            const sdkType = JSON.parse(fs.readFileSync(path.join(packageFolderPath, 'changelog-temp', 'package', 'package.json'), { encoding: 'utf-8' }))['sdk-type'];
             const clientType = getSDKType(packageFolderPath);
             if (sdkType && sdkType === 'mgmt' || clientType === SDKType.RestLevelClient) {
                 logger.info(`Package ${packageName} released before is track2 sdk.`);
                 logger.info('Start to generate changelog by comparing api.md.');
                 const npmPackageRoot = path.join(packageFolderPath, 'changelog-temp', 'package');
                 const apiMdFileNPM = getApiReviewPath(npmPackageRoot);
                 const apiMdFileLocal = getApiReviewPath(packageFolderPath);
+                const lastStableApiView: lastStableNpmViewParameter = {
+                    file: "ApiView",
+                    version: stableVersion,
+                    packageName: packageName,
+                    packageFolderPath: packageFolderPath,
+                    sdkRootPath: jsSdkRepoPath,
+                    npmPackagePath: npmPackageRoot,
+                }
+                if (!fs.existsSync(apiMdFileNPM)) {
+                    fs.mkdirSync(path.join(npmPackageRoot, "review"));
+                    await tryCreateLastStableNpmView(lastStableApiView);
```

Comment:
```suggestion
                    await tryCreateLastestStableNpmViewFromGithub(lastStableApiView);
```

Reply:
why it necessary here? looks like you already call it in tryReadNpmPackageChangelog